solidity/dashboard/src/reducers/liquidity-rewards.js

```diff
@@ -7,6 +7,7 @@ const liquidityPairInitialData = {
   reward: 0,
   wrappedTokenBalance: 0,
   lpBalance: 0,
+  lastNotificationRewardAmount: "0",
```

Comment:
Is that necessary? Maybe we can use the `reward` field?

Reply:
ahh I see https://github.com/keep-network/keep-core/pull/2272/files#diff-84df92fe28afc557a6f217094ff02bda2900ddadeac35d401f09d2a94b9e0f07R129

Reply:
Yea, so just to clarify we need to save that value as `lastNotificationRewardAmount` so after seven minutes we can compare it with the `reward` field and display the notification only when `reward` is higher. 

It will also prevent the notification to display when for example user get some rewards 3 minutes after last notification check and then withdraw all rewards. In this case we won't show the notification since the reward balance is now 0 and is lower than `lastNotificationRewardAmount` which was checked 7 minutes ago. We save the new value (which is now 0) in the variable for future comparisons.